### ISSUE TITLE
[Feature] Make text cursor able to land on blank space

### ISSUE THREAD

**Issue by miwg** [typora/typora-issues#5477]
When I create a new line or paragraph between two already existing lines, I would like my text cursor to land on this blank space and not automatically jump over it. I would also like to be able to move my cursor to a blank space in general which is not possible, unless I'm in source code mode.

I understand that not everybody would prefer the text cursor to behave in this way, but it is a common feature in editors in general, so an option for this would probably be appreciated by more people than me.

**Comment by abnerlee** [comment]
> When I create a new line or paragraph between two already existing lines, I would like my text cursor to land on this blank space and not automatically jump over it. 

would you provide a screencast that cursor  automatically jump over the empty paragraph?

**Comment by miwg** [comment]
Yes, of course!

The first video is from Typora, where I first create a paragraph between the lines and then try to target it with the text cursor.

https://user-images.githubusercontent.com/116036258/201494961-87015ff1-9b03-4271-ac0e-dc547fb01f53.mp4


In the second one I do the same, but in window's notepad. This is the behavior I expected.

https://user-images.githubusercontent.com/116036258/201494965-73745904-90fa-44bb-8105-b547eb1b2137.mp4

**Comment by miwg** [comment]
Yes, but what I was suggesting was adding the ability to manipulate space between the lines as it's done as in source code mode. 

I guess I could just use source code mode. Is there a way to apply Typora's standard color-and-font theme to the source code mode theme?
